pyproject.toml

```diff
@@ -112,19 +109,25 @@ line-length = 100
 target-version = ["py38", "py39", "py310", "py311"]
 preview = true
 
-[tool.isort]
-profile = "black"
+[tool.ruff]
+line-length = 100
+target-version = "py38"
+select = [
+    "E",  # pycodestyle
+    "F",  # pyflakes
+    "UP", # pyupgrade
+    "D",  # pydocstyle
+    "I",  # isort
+    "B",   # flake8-bugbear
+    "NPY"  # NumPy
+]
+ignore = ["D212"]
```

Comment:
Old version had hits config:
```
add-select = ["D213"]
add-ignore = ["D202", "D212"]
```

Reply:
"D213" - is automatically selected by "D".
"D202" - I fixed errors with it and removed the ignore